src/basic/in-addr-util.h

```diff
@@ -187,7 +177,7 @@ static inline int in_addr_prefix_from_string_auto(const char *p, int *ret_family
 }
 
 static inline size_t FAMILY_ADDRESS_SIZE(int family) {
-        assert(IN_SET(family, AF_INET, AF_INET6));
+        assert(family == AF_INET || family == AF_INET6);
```

Comment:
mehhh, I do not like this kind of change...
Maybe better to `switch()` with `assert_not_reached()` ?